template/python3-http-armhf/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM armhf/python:3.6-alpine
```

Comment:
The stock image 'library/python:3.6-alpine' is compatible with ARM.
So we can use 
```
FROM python:3.6-alpine
```
and if the image is build on ARM, the ARM version will be pulled.
Tested on Raspberry pi OK :+1:

Reply:
Oh I had no idea about that! You've tested it on the Raspberry Pi already?

Reply:
Yes I've tested and it ran successfully

Reply:
I just tried this and it failed. How were you able to run the image on your Raspberry Pi? Did you have to make any other changes?

Reply:
Tried what / what failed? Give steps please so we can try too.

Reply:
This appears to fail for me on armhf:

```
docker run -ti python:3.6-alpine /bin/sh
```

Reply:
I tried building the template with `python:3.6-alpine` and deploying it on the rpi, but the watchdog fails with the message `Error reading stdout: EOF`.